### PR TITLE
update kube and dependencies to v1.23

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

-->
```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,43 +15,85 @@ require (
...
-	k8s.io/client-go v0.22.0
-	k8s.io/component-base v0.22.0
-	k8s.io/klog/v2 v2.9.0
+	google.golang.org/grpc v1.40.0
+	k8s.io/api v0.23.0
+	k8s.io/apimachinery v0.23.0
+	k8s.io/client-go v0.23.0
+	k8s.io/component-base v0.23.0
+	k8s.io/klog/v2 v2.30.0
 	k8s.io/kubernetes v1.21.0
```
Comment (xing-yang) [inline go.mod:28]: Why is this still 1.21?

Comment (humblec) [reply go.mod]: it was a miss, updated this too. @xing-yang

**Pull request merged**